test/parallel/parallel.status

```diff
@@ -19,3 +19,5 @@ test-npm-install:      PASS,FLAKY
 
 [$system==freebsd]
 
+[$system==aix]
```

Comment:
If aix is fixed maybe we don't need this anymore?

Reply:
Probably not, so please go ahead and remove this.